peer_mentoring/groups/views.py

```diff
@@ -122,12 +96,48 @@ def group_show_post(request, group_id, post_id):
             comment.author = request.user
             comment.save()
             messages.success(request, "Comment Added!")
+            return redirect("groups:show_post", group_id, post_id)
     else:
         form = GroupPostCommentForm()
     context = {
         "group": group,
         "post": post,
-        "comment": comment,
+        "comments": comments,
         "form": form,
     }
     return render(request, "groups_show_post.html", context)
+
+
+@login_required
+def send_group_join_request(request, group_id):
+    group = get_object_or_404(Group, id=group_id)
+    if not GroupJoinRequest.objects.filter(user=request.user, group=group).exists():
+        GroupJoinRequest.objects.create(user=request.user, group=group)
```

Comment:
Some of this fixed here: https://github.com/JAWD-001/peer_mentoring/pull/48/files